src/nvim/main.c

```diff
@@ -133,6 +137,41 @@ static const char *err_extra_cmd =
   N_("Too many \"+command\", \"-c command\" or \"--cmd command\" arguments");
 
 
+void event_init(void)
+{
+  loop_init(&loop, NULL);
```

Comment:
This call to `loop_init()` is too late. `command_line_scan()` in `main()` could already need the loop initialized.
See https://github.com/neovim/neovim/issues/3045#issuecomment-123405833.